### PR TITLE
🚧 replace seed data with slack call

### PR DESCRIPTION
**Pull request by gmkohler**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
 
 @app.route('/emoji', methods=['GET'])
 def get_emoji_list():
+
     try:
-        with open('results.json') as json_file:
-            data = json.load(json_file)
-            return jsonify(data)
+
+        source_client = emoji_client(os.environ["SOURCE_SLACK_API_TOKEN"])
```
Comment (clanam) [inline app.py]: eventually we can pass this from the ui too! :D

File: emoji/emoji_transfer_service.py [hunk 2]
```diff
@@ -12,12 +22,8 @@ def transfer(source_client, destination_client, source_emoji_name):
     source_dict = source_client.emoji_dict()
     destination_dict = destination_client.emoji_dict()
 
-    emoji_path = source_dict.get(source_emoji_name)
+    emoji_path = url-url_even_if_alias(source_dict, source_emoji_name)
```
Comment (clanam) [inline emoji/emoji_transfer_service.py]: is this `url-url` bit a typo or a python thing?

Comment (gmkohler) [reply emoji/emoji_transfer_service.py]: typo, fix is up now

File: app.py [hunk 3]
```diff
@@ -10,11 +13,23 @@
...
+        source_dict = source_client.emoji_dict()
+
+        # map alias values to URL values or None
+        names_and_urls = {}
+        for k, v in source_dict.items():
+            names_and_urls[k] = url_even_if_alias(source_dict, k)
+
+        # filter out None values (aliases of standard emoji).
+        names_and_urls = { k: v for k, v in names_and_urls.items() if v is not None and not v.startswith("alias:") }
```
Comment (gmkohler) [inline app.py]: It seems like aliases of slack's built-in aliases stick around, pushing this up for now.  Fine to limit the UI options on selecting aliases, those are one-step processes in Slack's UI.

**Pull request merged**